### PR TITLE
[cloudDatabases] updates the FAQ documentation

### PR DESCRIPTION
**Pull request by ybriffa**
Updates the FAQ documentation for cloud databases

### CODE REVIEW

**Review by julienisapunk — APPROVED** [review]

File: pages/platform/databases/information_04_faq/guide.de-de.md [hunk 1]
```diff
@@ -204,23 +198,22 @@ We are based on UTC Time.
 
 To select the right amount of compute, RAM and storage, you will need to assess your needs.
 
-If you already have databases instances, you can check what is consumed today. For a new service, you will have to forecast you needs. Keep in mind that you can upgrade your DB instance range to a bigger one.
+If you already have databases instances, you can check what is consumed today. For a new service, you will have to forecast you needs. Keep in mind that you can scale up or down your DB.
```
Comment (thild42) [inline pages/platform/databases/information_04_faq/guide.de-de.md]: forecast you needs -> forecast your needs

File: pages/platform/databases/information_04_faq/guide.de-de.md [hunk 2]
```diff
@@ -375,6 +384,12 @@ Yes, based on the service plan, databases instances have the ability to connect
 It has to be specified and selected during database instance creation.
 If you select Private Network connectivity, you can not be connected through Public Network.
 
+### Can I move my instance from public to private network or vice versa ?
+
+No, you cannot. Once the networking type of an instance is set, you cannot switch to the other networking type.
+However, if your instance already has data and you want to move to the other networking type, you can
```
Comment (thild42) [inline pages/platform/databases/information_04_faq/guide.de-de.md]: remove carriage return at the end of this line.

File: pages/platform/databases/information_04_faq/guide.de-de.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 title: FAQ Public Cloud databases
 routes:
     canonical: '/pages/platform/databases/information_04_faq'
-updated: 2023-03-21
+updated: 2023-07-24
 ---
 
 **Last updated March 21<sup>st</sup>, 2023**
```
Comment (thild42) [inline pages/platform/databases/information_04_faq/guide.de-de.md]: This line could be removed as it is no more used by SNOW

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @ybriffa

**Pull request merged**